sass/_component_link.scss

```diff
@@ -12,4 +12,7 @@ a {
 	&:visited {
 		color: $link-hover-color;
 	}
+    
+    &:active {
+        color: $link-active-color;
```

Comment:
This fails to compile, there is a closing brace missing after this line

Reply:
Sorry😶

Reply:
No problem! Happens to all of us sometimes 😊